libs/sdk-backend-tiger/tests/integrated/catalog.test.ts

```diff
@@ -14,8 +14,9 @@ beforeAll(async () => {
 describe("tiger catalog", () => {
     it("should read catalog for reference workspace", async () => {
         const result = await backend.workspace(testWorkspace()).catalog().load();
-
-        expect(result).toMatchSnapshot();
+        const sanitizeResult = await sanitizeKeyWithNewValue(result, "uri", "sanitize_uri");
+        delete sanitizeResult.workspace;
```

Comment:
I'd introduce a  `sanitizeResult` or `sanitizeWorkspace` function, to do 2 steps

- `await sanitizeKeyWithNewValue(result, "uri", "sanitize_uri");`
- `delete sanitizeResult.workspace;`